src/tests/ftest/datamover/obj_small.py

```diff
@@ -130,7 +131,8 @@ def test_dm_obj_small_cont_clone(self):
         Test Description:
             DAOS-6858: Verify cloning a small container.
         :avocado: tags=all,daily_regression
-        :avocado: tags=datamover,cont_clone
+        :avocado: tags=vm
+        :avocado: tags=datamover,daos_cont_clone
```

Comment:
unrelated to this PR, but IMO daos_cont_close is a weird tag, and not sure if it's really beneficial

Reply:
Do you mean "clone" or "close"? I was trying to be explicit with `mfu_*` vs `daos_*` so it's more obvious which tests are daos tools vs mpifileutils.